cmd/web/main.go

```diff
@@ -111,8 +99,8 @@ func main() {
 		ReadHeaderTimeout: ServerTimeout,
 	}
 
-	infoLog.Println("Starting web server at " + app.cfg.addr)
-	errorLog.Fatalln(server.ListenAndServe())
+	log.Info().Msg("Starting web server at " + app.cfg.addr)
```

Comment:
It's good practice to use [contextual fields](https://github.com/rs/zerolog?tab=readme-ov-file#standard-types) when logging.
In this case we could add `.Str("address", app.cfg.addr).Msg("Starting web server")`